qulice-pmd/src/test/java/com/qulice/pmd/PMDValidatorTest.java

```diff
@@ -159,6 +159,23 @@ public void allowsStaticFieldInitializationWhenConstructorExists()
     }
 
     /**
+     * PMDValidator can forbid field initialization in several constructors.
+     * Only one constructor should do real work. Others - delegate to it.
+     * @throws Exception If something wrong happens inside.
+     */
+    @Test
+    public void forbidsFieldInitializationInSeveralConstructors()
+        throws Exception {
+        final String file = "FieldInitSeveralConstructors.java";
+        this.validatePMD(
+            file, false,
+            Matchers.containsString(
+                "(OnlyOneConstructorShouldDoInitialization)"
+            )
+        );
+    }
+
```

Comment:
@longtimeago let's add a test that should pass with a class that conforms to the new rule (1 init constructor, other constructors delegate to it).